tasks/config/esvm.js

```diff
@@ -62,6 +62,44 @@ module.exports = function (grunt) {
           }
         }
       }
+    },
+    withShield: {
```

Comment:
It seems like this is a shield+marvel setup. Would it be worthwhile to keep these separate? Or make separate tasks, one for shield, one for marvel? Or maybe not call this "withShield" but "withPlugins" or something like that?